### DISCUSSION TITLE
dependabot: gradle audit

### DISCUSSION THREAD

**Discussion by mcandre** [community/community#14097] [Code Security] [answered]
Now that Gradle has a decent plugins for reporting CVE's in JVM projects, would be helpful for dependabot to report these findings, just like it does for Ruby and Node projects.

See the net.ossindex.audit plugin, for example. This quickly identified several CVE's in my Java projects on GitHub that dependabot completely missed.

**Comment by jhutchings1** [comment, marked as answer]
We're working with the folks at Gradle on our [dependency submission API](https://github.com/github/roadmap/issues/467) so that we can get native support for Gradle analysis within the product. Keep an eye on the changelog for announcements in this space.

**Comment by mikepenz** [comment]
The [Dependency Submission API](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/using-the-dependency-submission-api) was launched as a public beta. 

Right now GitHub offers an official [go action](https://github.com/actions/go-dependency-submission/). 
Similar to the go action, we've built a gradle based one to publish gradle dependencies. The action is still very new, and all feedback is welcome. [Gradle Dependency Submission Action](https://github.com/mikepenz/gradle-dependency-submission)
